functions/src/sections/Section.controller.ts

```diff
@@ -50,16 +50,19 @@ export class SectionsController extends Controller {
               : 'TBA',
           days: days(m.meetingTime),
           where: m.meetingTime.buildingDescription
-            ? `${m.meetingTime.buildingDescription} ${m.meetingTime.room}`
+            ? `${formatBuilding(m.meetingTime.buildingDescription)} ${
+                m.meetingTime.room
+              }`
             : 'TBA',
           dateRange: `${m.meetingTime.startDate} - ${m.meetingTime.endDate}`,
           scheduleType: s.scheduleTypeDescription,
           instructors: s.faculty.map(
             (f) => f.displayName + `${f.primaryIndicator ? ' (P)' : ''}`
           ),
-          building: m.meetingTime.buildingDescription ?? undefined,
+          building:
+            formatBuilding(m.meetingTime.buildingDescription) ?? undefined,
           buildingAbbreviation: Buildings.get(
-            m.meetingTime.buildingDescription ?? ''
+            formatBuilding(m.meetingTime.buildingDescription) ?? ''
```

Comment:
Just want to double check you QA'd this change, if you add a course that has `&` to your timetable is the abbreviated version showing up in the timetable?

`Buildings.get` is accessing a mapping between the full name and the abbreviated name so since you're passing in something potentially different now with the `formatBuilding` function I just want to double check this